docs/usage/configuration-options.md

```diff
@@ -1603,6 +1603,21 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
+
+Use this field to restrict rules to a particular language. e.g.
```

Comment:
I think we need to elaborate a little more on why the option says categories but then we refer to languages